.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDatePeriodParser.cs

```diff
@@ -476,6 +477,12 @@ private DateTimeResolutionResult ParseBaseDatePeriod(string text, DateObject ref
                 innerResult = ParseDatePointWithAgoAndLater(text, referenceDate);
             }
 
+            // Cases like "for x weeks/days from today/12 sep etc."
+            if (!innerResult.Success)
+            {
+                innerResult = ParseDatePointWithForPrefix(text, referenceDate);
```

Comment:
Can this be moved to the english implementation if it exists ?

Reply:
As we talked, will consider it in next sprint.